lib/craco-antd.js

```diff
@@ -51,14 +51,19 @@ const overrideWebpackConfig = ({ context, webpackConfig, pluginOptions }) => {
   }
 
   const lessLoaderOptions = pluginOptions.lessLoaderOptions || {};
-  if (lessLoaderOptions.modifyVars) {
-    Object.assign(modifyVars, lessLoaderOptions.modifyVars);
+  if (
+    lessLoaderOptions.lessOptions &&
+    lessLoaderOptions.lessOptions.modifyVars
+  ) {
+    Object.assign(modifyVars, lessLoaderOptions.lessOptions.modifyVars);
   }
 
-  lessLoaderOptions.modifyVars = modifyVars;
-  // javascriptEnabled: true is suggested in the Ant Design docs:
-  // https://ant.design/docs/react/customize-theme#Customize-in-webpack
-  lessLoaderOptions.javascriptEnabled = true;
+  lessLoaderOptions.lessOptions = {
+    modifyVars: modifyVars,
+    // javascriptEnabled: true is suggested in the Ant Design docs:
+    // https://ant.design/docs/react/customize-theme#Customize-in-webpack
+    javascriptEnabled: true
+  };
```

Comment:
This is dangerous since it overrides `lessOptions` and passes only `modifyVars` and `javascriptEnabled`

Reply:
@ndbroadbent

Reply:
Ah, good point. I will revert this commit now and wait for a PR that solves this in a better way.

Reply:
Thanks for the review!
https://github.com/DocSpring/craco-antd/pull/38